src/jit/lclvars.cpp

```diff
@@ -353,8 +353,9 @@ void                Compiler::lvaInitArgs(InitVarDscInfo *          varDscInfo)
     assert (varDscInfo->intRegArgNum <= MAX_REG_ARG);
 
     codeGen->intRegState.rsCalleeRegArgNum = varDscInfo->intRegArgNum;
-
-#if !FEATURE_STACK_FP_X87
+#ifdef ARM_SOFTFP
+    codeGen->intRegState.rsCalleeRegArgNum += varDscInfo->floatRegArgNum;
+#elif !FEATURE_STACK_FP_X87
     codeGen->floatRegState.rsCalleeRegArgNum = varDscInfo->floatRegArgNum;
```

Comment:
I don't think this change is necessary, since floatRegArgNum should always be zero for softFP